public/video-ui/src/components/EditSaveCancel/index.js

```diff
@@ -36,11 +37,11 @@ class EditSaveCancel extends React.Component {
   }
 
   renderCancelButton() {
-    const { onCancel } = this.props;
+    const { canCancel, onCancel } = this.props;
 
     return (
-      <button onClick={onCancel}>
-        <Icon icon="cancel" className="icon__cancel">Cancel</Icon>
+      <button onClick={onCancel} disabled={canCancel ? !canCancel() : false}>
+        <Icon icon="cancel" className={`icon__cancel ${canCancel ? (canCancel() ? '' : 'disabled') : ''}`}>Cancel</Icon>
```

Comment:
Maybe refactor the className generation out? It's hard to read the ternary interpolated into the string